.github/workflows/pytest-cov.yaml

```diff
@@ -0,0 +1,43 @@
+name: Pytest Coverage
+on:
+  push:
+    branches: [develop, 53-create-git-actions-workflow-pytest-cov]
+  pull_request:
+    branches: [develop]
+
+jobs:
+  run_pytest_coverage:
+    name: Run Pytest Coverage
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.10"
+
+      - name: Install App
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .[dev]
+
+      - name: Run pytest-cov
+        run: |
+          pytest --cov-fail-under=5 --cov-report=html:docs --cov=uwtools
+
+      - name: Upload Reports
+        uses: actions/upload-artifact@v3
+        with:
+          name: coverage-report
+          path: docs
+
+      - name: Upload to pages
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git add -f .
+          git commit -m "update code coverage"
+          git push
```

Comment:
I don't this is necessary if we use the action from the marketplace. 
Also, gh_pages should only contain the coverage, not the entire repo.